packages/core/content-manager/admin/src/utils/validation.ts

```diff
@@ -199,7 +216,7 @@ const addRequiredValidation: ValidationFn = (attribute) => (schema) => {
     });
   }
 
-  return schema.nullable();
+  return schema?.nullable ? schema.nullable() : schema;
```

Comment:
Why would a schema not be nullable?

Reply:
i'd write a comment it's not clear and feels odd to me on first glance, someone will need to know why you've made the change :)